python/graphstorm/model/loss_func.py

```diff
@@ -71,6 +71,73 @@ def out_dims(self):
         """
         return None
 
+class FocalLossFunc(GSLayer):
+    """ Focal loss function for classification.
+
+    Copy from torchvision.ops.sigmoid_focal_loss.
+    Only with mean reduction.
+    See more details on https://pytorch.org/vision/main/_modules/torchvision/ops/focal_loss.html.
+
+    Parameters
+    ----------
+    alpha: float
+        Weighting factor in range (0,1) to balance
+        ositive vs negative examples or -1 for ignore. Default: ``0.25``.
+    gamma: float
+        Exponent of the modulating factor (1 - p_t) to
+        balance easy vs hard examples. Default: ``2``.
+    """
+    def __init__(self, alpha=0.25, gamma=2):
```

Comment:
The default alpha and gamma are set in gsf.py, do we need to set them here as well?

Reply:
I just keep the default value here.